src/icons/Icons.stories.tsx

```diff
@@ -0,0 +1,25 @@
+import { SvgIcon } from '@mui/material';
+import { ComponentMeta, ComponentStory } from '@storybook/react';
+
+import * as icons from './';
+
+export default {
+  title: 'Icons/SvgIcon',
+  component: SvgIcon,
+  argTypes: {
+    component: {
+      options: Object.keys(icons),
+      mapping: icons,
+      control: { type: 'select' },
+    },
+  },
+} as ComponentMeta<typeof SvgIcon>;
+
+const Template: ComponentStory<typeof SvgIcon> = (args) => (
+  <SvgIcon {...args} inheritViewBox />
```

Comment:
Not sure if that's much better haha but it would be nice to remove the dependency on `SvgIcon` from `@mui/material`.

In theory, (and I am not sure if this is something we exactly are reaching for but something I think of when making a design system on top of another system) if we want to be self-dependent and users of our system never need to install `@mui/material` as a direct dependency, then we should not reference `@mui/material` anywhere in storybook either.

Reply:
We're using Material UI as a peerDependency, so users of this library will need to include their own compatible version of Material UI to use this library. 

The SVG files actually render SvgIcons in their generated components to reduce some boilerplate. SvgIcon will set a 24x24 viewBox, while our SVGs are drawn in a 16x16 viewBox, so they aren't compatible unless `inheritViewBox` is set, something we and library consumers will have to remember to set every time. The alternative would be to have our own SvgIcon component that wraps SvgIcon with inheritViewBox set, but I didn't do this because there might be confusion between the two components. I wrapped them in the story because I initially didn't have a custom template, but it remained compatible so I didn't change anything. I did just notice an issue: I didn't set `inheritViewBox` in the template, and didn't catch it because that parameter is correctly set here. I'll do as you suggest here, so we can view the component exactly as it's generated so this error can be better caught.

Reply:
> I did just notice an issue: I didn't set inheritViewBox in the template, and didn't catch it because that parameter is correctly set here. I'll do as you suggest here, so we can view the component exactly as it's generated so this error can be better caught.

I am wrong about this. When the SvgIcon is generated, it passes the SVG markup directly into its children and it sets all attributes from the original SVG file onto SvgIcon, including the viewBox. I confirmed this by changing the `Icons.stories.tsx` file to render the icons directly now.